src/infra/HttpRequester.ts

```diff
@@ -21,4 +21,17 @@ export class HttpRequester implements HttpRequesterService {
 
     return fetch(url, fetchInit);
   }
+
+  get(url: string): Promise<Response> {
+    const fetchInit = {
+      method: 'GET',
+      mode: 'no-cors' as RequestMode,
+    };
+
+    if (typeof (window) !== 'undefined' && window.fetch) {
+      return window.fetch(url, fetchInit);
```

Comment:
I copied this from the existing implementation. @tmeyer2115 ?

Reply:
This is still Node compatible. The `window` references are purely to see if this is being used within a browser. If `window` is undefined (i.e. we're in a Node app), then we use `fetch` instead of `window.fetch`. I wouldn't worry for now. Down the road, we should use something like cross fetch where we don't need this conditional logic.